Add docs for release of remote cube (#2254)
SkylineCommunications/dataminer-docs: user-guide/Cloud_Platform/RemoteAccess/Cloud_Remote_Access.md

> [!NOTE]
> To access the web apps, you will need to have the [General > DataMiner web apps](xref:DataMiner_user_permissions#general--dataminer-web-apps--dataminer-cube-mobile-access) user permission, as well as any other user permissions required to access specific apps.

## Accessing your DMS remotely with Cube

> If [remote access is enabled](xref:Controlling_remote_access) for a DMS, and you have been [granted access to dataminer.services features](xref:Giving_users_access_to_cloud_features), you can access your DMS via the remote access URL. To be able to remotely access your system with Cube you need to have the Cube desktop application installed on your system. More information on how to install Cube can be found (here)[xref:Installing_configuring_the_DataMiner_Cube_software].

Reviewer: Not sure if we want to capitalize Remote Access?
```suggestion
If [Remote Access is enabled](xref:Controlling_remote_access) for a DMS, and you have been [granted access to dataminer.services features](xref:Giving_users_access_to_cloud_features), you can access your DMS via the Remote Access URL. To be able to remotely access your system with Cube you need to have the Cube desktop application installed on your system. More information on how to install Cube can be found (here)[xref:Installing_configuring_the_DataMiner_Cube_software].
```

Remote Cube will use the same URL that is being used for remote access to the web pages, but without the protocol prefix "https://".

> To access your system remotely with Cube their are a few options to start the Cube session:

Reviewer: ```suggestion
To access your system remotely with Cube there are a few options to start the Cube session:
```

### By using the Cube launcher

> Open the Cube launcher and add a new DMS. Here you will be prompted to enter a "host" and an "alias". In the host field you need to fill in the Remote Access URL of your system, without the protocol prefix. This will look something like this: "ziine-skyline.on.dataminer.services". To be able to easily identify your system you can fill in a human readable and easy to remember name in the alias field.

Reviewer: To align with the image:
```suggestion
Open the Cube launcher and add a new DMS. Here you will be prompted to enter a "host" and an "alias". In the host field you need to fill in the Remote Access URL of your system, without the protocol prefix. This will look something like this: "ziine-ziine.on.dataminer.services". To be able to easily identify your system you can fill in a human readable and easy to remember name in the alias field.
```

Author: Will adjust the image to keep it consistent with the rest of the doc.

> Open the Cube launcher and add a new DMS. Here you will be prompted to enter a "host" and an "alias". In the host field you need to fill in the Remote Access URL of your system, without the protocol prefix. This will look something like this: "ziine-skyline.on.dataminer.services". To be able to easily identify your system you can fill in a human readable and easy to remember name in the alias field.

Reviewer: Not sure if required, but I feel like "field" is missing here
```suggestion
Open the Cube launcher and add a new DMS. Here you will be prompted to enter a "host" and an "alias" field. In the host field you need to fill in the Remote Access URL of your system, without the protocol prefix. This will look something like this: "ziine-skyline.on.dataminer.services". To be able to easily identify your system you can fill in a human readable and easy to remember name in the alias field.
```

Author: Will adjust the text.

> ![Remote cube in the launcher](~/user-guide/images/RemoteCubeLauncher.png)

Reviewer: ```suggestion
![Remote Cube in the launcher](~/user-guide/images/RemoteCubeLauncher.png)
```

### By using the button on admin.dataminer.services

> Open a browser and navigate to [admin.dataminer.services](https://admin.dataminer.services) and navigate to the overview of your system. Here you will find a button that is labeled "Open Desktop Application". Pressing this button will open up the Cube application with the correct URL in the host field. Simply click on "Connect" and Cube will start a session with your DMS.

Reviewer: Wouldn't repeat navigate twice
```suggestion
Open a browser and navigate to [admin.dataminer.services](https://admin.dataminer.services) and go to the overview of your system. Here you will find a button that is labeled "Open Desktop Application". Pressing this button will open up the Cube application with the correct URL in the host field already filled in. Simply click on "Connect" and Cube will start a session with your DMS.
```

> Open a browser and navigate to [admin.dataminer.services](https://admin.dataminer.services) and navigate to the overview of your system. Here you will find a button that is labeled "Open Desktop Application". Pressing this button will open up the Cube application with the correct URL in the host field. Simply click the "connect" button and cube will setup a session with your DMS.

Reviewer: ```suggestion
Open a browser and navigate to [admin.dataminer.services](https://admin.dataminer.services) and navigate to the overview of your system. Here you will find a button that is labeled "Open Desktop Application". Pressing this button will open up the Cube application with the correct URL in the host field. Simply click on "Connect" and Cube will start a session with your DMS.
```

> ![Remote cube in the admin app](~/user-guide/images/RemoteCubeAdminApp.png)

Reviewer: ```suggestion
![Remote Cube in the admin app](~/user-guide/images/RemoteCubeAdminApp.png)
```

### By using the button on dataminer.services

> Open a browser and navigate to [dataminer.services](https://dataminer.services). Find your DMS in the list and click the "Open  in desktop app" button. Pressing this button will open up the Cube application with the correct URL in the host field. Simply click "Connect" and Cube will setup a session with your DMS.

Reviewer: ```suggestion
Open a browser and navigate to [dataminer.services](https://dataminer.services). Find your DMS in the list and click the "Open  in desktop app" button. Pressing this button will open up the Cube application with the correct URL in the host field already filled in. Simply click "Connect" and Cube will setup a session with your DMS.
```

> Open a browser and navigate to [dataminer.services](https://dataminer.services). Find your DMS in the list and click the "Open desktop application" button. Pressing this button will open up the Cube application with the correct URL in the host field. Simply click the "connect" button and cube will setup a session with your DMS.

Reviewer: ```suggestion
Open a browser and navigate to [dataminer.services](https://dataminer.services). Find your DMS in the list and click on "Open desktop in application". Pressing this button will open up the Cube application with the correct URL in the host field. Simply click the "Connect" button and Cube will start a session with your DMS.
```

> ![Remote cube in the home app](~/user-guide/images/RemoteCubeHomeApp.png)

Reviewer: ```suggestion
![Remote Cube in the home app](~/user-guide/images/RemoteCubeHomeApp.png)
```
